Sprint-3/implement/test/is-valid-traingle.test.js

```diff
@@ -0,0 +1,16 @@
+const isValidTriangle = require("../is-valid-triangle");
+
+test("Is a valid triangle", () => {
+  expect(isValidTriangle(3, 3, 3)).toBe(true);
+});
+
+test("Invalid Triangle with Negative", () => {
+  expect(isValidTriangle(1, -2, 1)).toBe(false);
+});
+
+test("Invalid checking triangle with zero sides", () => {
```

Comment:
Your code will fail this test because it doesn't check for zero side lengths.

Reply:
i just add to >0 to check negative